### PR TITLE
Support for CC2020

### PR DESCRIPTION
**Pull request by Lilipi**
Hello,
You could find here a PR for CC2020 products support.
Thanks

### CODE REVIEW

**Comment by pcdeshmukh** [comment]
Cool... thanks Lilipi
I tried changes with InDesign & InCopy on Windows it worked fine for me.

File: tasks/lib/hosts.json [hunk 1]
```diff
@@ -1,5 +1,79 @@
...
+            "version": { "min": "24.0", "max": "24.9" },
+            "bin": { "win": "Support Files/Contents/Windows/Illustrator.exe", "mac": "Adobe Illustrator CC 2020.app" },
+            "x64": true
+        },
+        "indesign": {
+            "familyname": "InDesign",
+            "name": "InDesign",
+            "ids": [ "IDSN" ],
+            "version": { "min": "15.0", "max": "15.9" },
+            "bin": { "win": "InDesign.exe", "mac": "Adobe InDesign CC 2020.app" },
```
Comment (pcdeshmukh) [inline tasks/lib/hosts.json]: Name needs change for mac it is - "Adobe InDesign 2020". 
Same is the case with InCopy it is - "Adobe InCopy 2020"

**Comment by Lilipi** [comment]
Hi,
Thanks for your feedback. I made corrections.
Don't you think all binaries have been renamed on Mac without "CC" in their names (not only InDesign and InCopy) ?

**Comment by pcdeshmukh** [comment]
> Hi,
> Thanks for your feedback. I made corrections.
> Don't you think all binaries have been renamed on Mac without "CC" in their names (not only InDesign and InCopy) ?

Yes I agree. Alessandro made similar change in another pull request.
I have access to only these two apps hence I can not verify it for other CC apps.

**Comment by fcamarlinghi** [comment]
Hey, thanks for the PR! Yes, it seems Adobe dropped the CC in all app names in 2020. I'm going to integrate your PR with #39  and have a new version of grunt-cep out later today. Thanks again.

**Pull request closed without merging**